### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by jbradl11**
Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

Updated README, added LICENSE and notice.md file

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,56 @@
-bonnie
-======
+[![Build Status](https://travis-ci.org/projecttacoma/bonnie.svg?branch=master)](https://travis-ci.org/projecttacoma/bonnie)
```
Comment (lukoktonos) [inline README.md]: This should be travis-ci.com

**Review by lukoktonos — DISMISSED** [review]

**Review by lukoktonos — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,10 +1,54 @@
...
+![GitHub version](https://badge.fury.io/gh/projecttacoma%2Fbonnie.svg)
 
-Clinical Quality Measure Testing Tool
 
-License
-======
+
+# Bonnie
+
+Bonnie is designed to support streamlined and efficient pre-testing of eCQMs, particularly those used in the Centers for Medicare & Medicaid Services (CMS) quality reporting programs.
```
Comment (edeyoung) [inline README.md]: i think bonnie deserves more description.
can you use the intro to the user guide?
what about qrda exporting? that covers another user base

File: README.md [hunk 3]
```diff
@@ -1,10 +1,54 @@
...
+```
+
+### To run backend Ruby tests
+
+```
+bundle exec rake test
+```
+
+
+## Issues Reporting
```
Comment (edeyoung) [inline README.md]:
```suggestion
## Reporting Issues
```

**Review by edeyoung — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (edeyoung) [inline README.md]:
```suggestion
Bonnie is a software tool that allows electronic clinical quality measure (eCQM) developers to test and verify the behavior of their eCQM logic. The main goal of the Bonnie application is to reduce the number of defects in eCQMs by providing a robust and automated testing framework. The Bonnie application allows measure developers to independently load measures that they have constructed using the Measure Authoring Tool (MAT). Developers can then use the measure metadata to build a synthetic patient test deck for the measure from the clinical elements defined during the measure construction process. By using measure metadata as a basis for building synthetic patients, developers can quickly and efficiently create a test deck for a measure. The Bonnie application helps measure developers execute the measure logic against the constructed patient test deck and evaluate whether the logic aligns with the intent of the measure. Additionally, Bonnie allows users to export their synthetic patient test deck in the Quality Reporting Document Architecture (QRDA) Category 1 data standard to enable them to more easily test their own systems. 

Bonnie has been designed to integrate with the nationally recognized data standards the Centers for Medicare & Medicaid Services (CMS) quality reporting programs use for expressing eCQM logic for machine-to-machine interoperability. This integration provides enormous value to the eCQM program and federal policy leaders and stakeholders. The Bonnie tool verifies that the new and evolving standards for eCQMs used in the CMS quality reporting programs are flexible and can be implemented in software.
```

Comment (edeyoung) [reply README.md]: ewww... copy/pasted and didn't realize the returns were included. Will remove.

**Review by edeyoung — APPROVED** [review]

**Review by lukoktonos — APPROVED** [review]

**Pull request merged**